### PR TITLE
Avoid import Imputer error for different sklearn verions

### LINKED ISSUE #430: Imputer was deprecated in version 0.20 and will be removed in 0.22. Import impute.SimpleImputer from sklearn instead.
/kind bug

**What steps did you take and what happened:**
[A clear and concise description of what the bug is.]
Following the steps in https://github.com/kubeflow/fairing/tree/master/examples/prediction
Error appeard during import Imputer

>>> sklearn.__version__
'0.22'
>>> from sklearn.preprocessing import Imputer
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'Imputer' from 'sklearn.preprocessing' (/Users/fengli/go/src/github.com/kubeflow/venv/lib/python3.8/site-packages/sklearn/preprocessing/__init__.py)

**What did you expect to happen:**
No Error

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]
Search from Google,  Imputer was deprecated in version 0.20 and will be removed in 0.22. Import impute.SimpleImputer from sklearn instead.

**Environment:**

- Fairing version: (use `python -c "import kubeflow.fairing; print(kubeflow.fairing.__version__)"`):
- Kubeflow version: (version number can be found at the bottom left corner of the Kubeflow dashboard):
- Minikube version:
- Kubernetes version: (use `kubectl version`):
- OS (e.g. from `/etc/os-release`):

**NOTE**: If you are using fair from master, please provide us the git commit hash.

### PR DESCRIPTION
**Pull request by fenglixa**
**What this PR does / why we need it**:
Following the steps in https://github.com/kubeflow/fairing/tree/master/examples/prediction
Error will be as below:

>sklearn.version
'0.22'
from sklearn.preprocessing import Imputer
Traceback (most recent call last):
File "", line 1, in
ImportError: cannot import name 'Imputer' from 'sklearn.preprocessing' (/Users/fengli/go/src/github.com/kubeflow/venv/lib/python3.8/site-packages/sklearn/preprocessing/init.py)

After investigate, Imputer was deprecated in version 0.20 and will be removed in 0.22. Import impute.SimpleImputer from sklearn instead

This fix will be ensure import Imputer sucessfully for different sklearn verions

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #430

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note
NONE
```

UT: PASS, Follow the steps in prediction could be sucessfully now.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/fairing/431)

### CODE REVIEW

**Comment by fenglixa** [comment]
\\cc @jinchihe ,  coule you help review, comments? Thanks!

File: examples/notebook/deploy-to-gcp.ipynb [hunk 1]
```diff
@@ -237,4 +240,4 @@
  },
  "nbformat": 4,
  "nbformat_minor": 2
-}
+}
```
Comment (jinchihe) [inline examples/notebook/deploy-to-gcp.ipynb]: nit: Add new line. Same below.. Thanks

File: examples/prediction/HousingTrain.py [hunk 2]
```diff
@@ -18,8 +18,11 @@
 import pandas as pd
 from sklearn.metrics import mean_absolute_error
 from sklearn.model_selection import train_test_split
-from sklearn.preprocessing import Imputer
 from xgboost import XGBRegressor
+try:
+    from sklearn.preprocessing import Imputer
+except ImportError:
+     from sklearn.impute import SimpleImputer as Imputer
```
Comment (abhi-g) [inline examples/prediction/HousingTrain.py]: please fix indentation here at line 25. Its generating lint error as well.

File: examples/notebook/deploy-to-gcp.ipynb (hunk 1 above)
Comment (fenglixa) [reply examples/notebook/deploy-to-gcp.ipynb]: Thanks, updated, seems it was caused by my vscode auto deleted it.

File: examples/prediction/HousingTrain.py (hunk 2 above)
Comment (fenglixa) [reply examples/prediction/HousingTrain.py]: Thanks for reivew, updated it, sorry for my typo.

**Comment by fenglixa** [comment]
Get test error for test_lightgbm, should be releated to another issue of #425
```
Step #2: tests/integration/gcp/test_running_in_notebooks.py::test_lightgbm
Step #2: [gw4] [100%] FAILED tests/integration/gcp/test_running_in_notebooks.py::test_lightgbm WARNING: Failed to generate report: No data to report.
Step #2:
Step #2:
Step #2:
Step #2: =================================== FAILURES ===================================
Step #2: ________________________________ test_lightgbm _________________________________
Step #2: [gw4] linux -- Python 3.7.5 /usr/local/bin/python
Step #2:
Step #2:     def test_lightgbm():
Step #2:         file_dir = os.path.dirname(__file__)
Step #2:         notebook_rel_path = "../../../examples/lightgbm/distributed-training.ipynb"
Step #2:         notebook_abs_path = os.path.normpath(
Step #2:             os.path.join(file_dir, notebook_rel_path))
Step #2:         expected_messages = [
Step #2:             "Copying gs://fairing-lightgbm/regression-example/regression.train.weight",
Step #2:             "[LightGBM] [Info] Finished initializing network",  # dist training setup
Step #2:             "[LightGBM] [Info] Iteration:10, valid_1 l2 : 0.2",
Step #2:             "[LightGBM] [Info] Finished training",
Step #2:             "Prediction mean: 0.5",
Step #2:             ", count: 500"
Step #2:         ]
Step #2: >       run_notebook_test(notebook_abs_path, expected_messages)
Step #2:
Step #2: tests/integration/gcp/test_running_in_notebooks.py:50:
```

/retest

**Pull request merged**